src/components/Sessions/SessionExpired.js

```diff
@@ -0,0 +1,50 @@
+import React, {useContext, useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/Errors/ErrorMessage';
+import Link from 'components/Link';
+import useSessionTimeout from 'hooks/useSessionTimeout';
+
+import {ConfigContext} from '../../Context';
+import {flagNoActiveSubmission} from '../../data/submissions';
```

Comment:
```suggestion
import {ConfigContext} from 'Context';
import {flagNoActiveSubmission} from 'data/submissions';
```
we try to avoid "path hell" :smile: